colossus-examples/src/main/scala/colossus-examples/WebsocketExample.scala

```diff
@@ -82,13 +83,17 @@ object WebsocketExample {
           case "MANY" => {
```

Comment:
Is there one proxy actor per connection? It seems that handling "MANY" will be blocking until count equals zero so if the client tries to send another message (like "EXIT) it wouldn't be handled immediately. At least in this example, the logic is the same in OK and FULL state. Is there a scenario where we would want a different behaviour if full?